src/HttpClient.js

```diff
@@ -49,9 +49,10 @@ class HttpClient {
     headers={},
     attempts=0,
     failover=true,
-    forceFailover=false
+    forceFailover=false,
+    nodeUrl
   }) {
-    let baseURI = this.BaseURI();
+    let baseURI = this.BaseURI(nodeUrl);
```

Comment:
Need to prevent failover if nodeUrl is passed in - line 109 maybe:

`if(!writeToken && !nodeUrl && ((failover && parseInt(response.status) >= 500) || forceFailover) && attempts < this.uris.length)`